packages/inter-protocol/src/vaultFactory/liquidateIncrementally.js

```diff
@@ -48,7 +48,7 @@ const trace = makeTracer('LiqI', false);
 
 /**
  * @typedef {{
- *   amm: XYKAMMPublicFacet,
+ *   amm: any,
```

Comment:
consider a `TODO` comment linking the the issue or PR that will remove AMM entirely

Reply:
Is it important to use `any` here? Can we [use unknown instead](https://github.com/Agoric/agoric-sdk/wiki/Types-documentation#prefer-unknown-to-any)?